### ISSUE TITLE
Can't set high refresh rate

### ISSUE THREAD

**Issue by luigir-it** [hyprwm/Hyprland#4764]
### Hyprland Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch HEAD at commit 84ab8d11e8951a6551d1e1bf87796a8589da6d47  (props: bump ver to 0.35.0).
Date: Mon Feb 5 01:59:02 2024
Tag: v0.35.0

flags: (if any)

```

</details>

### Bug or Regression?

Bug

### Description

I have a monitor that supports up to 3440x1440@**165**, but the maximum shown by wlr-randr is 3440x1440@**100**

I tried booting with a live image of `arcolinux-hyprland-v24` and `garuda-hyprland` but I have the same problem

I did the same with `ubuntu sway 23.10`  and there wlr-randr showed the right refresh rate
I finally booted again  `garuda-hyprland` , this time using nouveau drivers, and it also showed the right refresh rate

So I don't know if this is actually an Hyprland issue... Because it seems to work with nouveau. The last check I should probably do is whether `ubuntu sway 23.10`  was using proprietary drivers or not.

### How to reproduce

Monitor: AW3423DWF

**Comment by vaxerski** [comment]
if anything it's a wlroots issue that stems from nvidia's poop drivers

**Comment by alba4k** [comment]
@luigir-it 

Unofficial solution, sure, but try overclocking your monitor by using
`monitor = IDK, 1265.25 3440 3752 4136 4832 1440 1443 1453 1588 -hsync +vsync, 0x0, 1`

Of course, adjust the port name, position and scaling as needed

![immagine](https://github.com/hyprwm/Hyprland/assets/84153269/08928ede-ac04-45de-94aa-d0795ad670e5)

**Comment by luigir-it** [comment]
> Unofficial solution, sure, but try "overclocking" your monitor by using `monitor = IDK, 1265.25 3440 3752 4136 4832 1440 1443 1453 1588 -hsync +vsync, 0x0, 1`

I tried but unfortunately doesn't work. 

Meanwhile, I tried also an X11 session with plasma and proprietary drivers, and it's still capped at 100hz. So I can confirm it's not an hyprland issue. I opened an issue over [nvidia website](https://forums.developer.nvidia.com/t/many-refresh-rate-missing-using-nvidia-545-drivers-on-alienware-aw3423dwf/283917).
